paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -334,6 +366,33 @@
                     "type": "number",
                     "minimum": 0.0,
                     "exclusiveMinimum": false
+                },
+                "host_port": {
+                    "type": "integer",
+                    "default": 0,
+                    "minimum": 0,
+                    "maximum": 65535,
+                    "exclusiveMinimum": false
+                },
+                "dependencies_reference": {
+                    "type": "string"
+                },
+                "extra_docker_args": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    }
+                },
```

Comment:
k8s doesn't support extra docker args

Reply:
Yeah, the point for this change is to allow user (or us) using `git mv` for marathon-{cluster}.yaml to kubernetes-{cluster}.yaml and clean up afterward.